### PR TITLE
Renaming nodeJs CI/CD templates.

### PR DESCRIPTION
**Pull request by ajinkya599**
- Renaming node.js CI/CD templates.
- Changing names and descriptions accordingly.

### CODE REVIEW

File: templates/node.js-functionapp-to-azure-on-linux.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Node.js Function App on Linux
-# Build and deploy a Node.js Function app on Linux.
+# Node.js Function App to Azure on Linux
+# Build a Node.js Function app and deploy it to Azure on Linux.
```
Comment (bansalaseem) [inline templates/node.js-functionapp-to-azure-on-linux.yml]: deploy it to Azure as a Linux function app

File: templates/node.js-express-webapp-to-azure-on-linux.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# Node.js Express Web App on Linux
-# Build a Node.js Express app and deploy it as a Linux web app.
+# Node.js Express Web App to Azure on Linux
```
Comment (bansalaseem) [inline templates/node.js-express-webapp-to-azure-on-linux.yml]: to Linux on Azure

Please do it everywhere

Comment (ajinkya599) [reply templates/node.js-express-webapp-to-azure-on-linux.yml]: Done

File: templates/node.js-functionapp-to-azure-on-linux.yml (hunk 1 above)
Comment (ajinkya599) [reply templates/node.js-functionapp-to-azure-on-linux.yml]: done

**Pull request merged**